tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids_test.py

```diff
@@ -0,0 +1,101 @@
+# Python imports
+import unittest
+
+# Third party imports
+import mock
+from mock import patch
+import pandas as pd
+
+# Project imports
+from cdr_cleaner.cleaning_rules import sandbox_and_remove_pids
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+
+class SandboxAndRemovePidsTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'project_id'
+        self.dataset_id = 'dataset_id'
+        self.sandbox_dataset = 'sandbox_dataset'
+        self.ticket_number = 'DC_XXX'
+        self.pids = [
+            324264993, 307753491, 335484227, 338965846, 354812933, 324983298,
+            366423185, 352721597, 352775367, 314281264, 319123185, 325306942,
+            324518105, 320577401, 339641873, 329210551, 364674103, 339564778,
+            309381334, 352068257, 353001073, 319604059, 336744297, 357830316,
+            352653514, 349988031, 349731310, 359249014, 361359486, 315083772,
+            358741126, 312045923, 313427389, 341366267, 305170199, 308597253,
+            348834424, 325536292, 360363123
+        ]
+        self.person_table_list = [
+            'observation', 'drug_era', 'observation_period', '_deid_map',
+            'visit_occurrence', 'condition_era', 'measurement', 'person',
+            'procedure_occurrence', 'specimen', 'death', 'device_exposure',
+            'payer_plan_period', '_ehr_consent', 'condition_occurrence',
+            'drug_exposure', 'note', 'dose_era'
+        ]
+
+        mock_bq_query_patcher = patch(
```

Comment:
We probably need a different way for mocking the return_value of `bq_util.query`, please check the example https://github.com/all-of-us/curation/blob/3d25aa64d3d76132d0c4289aecd186081e5f023f/tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/validate_missing_participant_test.py#L163

Reply:
Since this is calling the client library I needed to call the entire function so that the Client would also be mocked.